sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareOperation.cs

```diff
@@ -206,17 +222,23 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
                     {
                         // we need to first assign a vaue and then mark the operation as completed to avoid race conditions
                         _value = Transforms.ConvertToRecognizeHealthcareEntitiesResultCollection(update.Value.Results, _idToIndexMap);
-
```

Comment:
to simplify this code you could do:
```
_response = update.GetRawResponse();
_status = update.Value.Status;
if (_status  == JobStatus.Succeeded)
...
```

That way you only assign the value once and it represents what the service sent. It is less error prune